src/Components/LoginSignUp/SignUp.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import {Header} from "../Header/Header";
+// import "./Login.css"
+import { useAuth } from "../../context/Authentication/LoginContext";
+
+const Signup = () => {
+  const { signupHandler } = useAuth();
+
+  const [data, setData] = useState({
+    name: "",
+    email: "",
+    password: "",
+    confirmPassword: "",
+  });
+
+  const submitHandler = (e) => {
+    e.preventDefault();
+    if (data.password === data.confirmPassword) {
+      console.log(data);
+    } else {
+      console.log("passwords are not same ");
+    }
+  };
+
+  return (
+    <>
+     <Header />
```

Comment:
i see header component is reused multiple times in different pages, you can move it to App so that its global and dont have to copy it in all pages